internal/app/auth_server.go

```diff
@@ -18,6 +18,12 @@ import (
 	"github.com/golang/gddo/httputil/header"
 )
 
+var (
+	loginEndpoint  = "api/login"
+	logoutEndpoint = "v2/logout"
```

Comment:
```suggestion
	logoutEndpoint = "api/logout"
```

Reply:
@rot1024 
I made the logout endpoint like this because I could not find a way to change it in `auth0/auth0-react` lib, so should I leave it as it is, or change it to `api/logout`

Reply:
I see. It is hard-coded: https://github.com/auth0/auth0-spa-js/blob/master/src/Auth0Client.ts#L976

So I suggest create 2 endpoints that use the same handler: one is `api/logout` (for general) and another is `v2/logout` (for Auth0)